pkg/destroy/aws/shared.go

```diff
@@ -117,6 +117,12 @@ func (o *ClusterUninstaller) removeSharedTag(ctx context.Context, session *sessi
 				}
 				result, err := tagClient.UntagResourcesWithContext(ctx, request)
 				if err != nil {
+					if aerr, ok := err.(awserr.Error); ok {
```

Comment:
We should probably be using `errors.As` so typecasting can work even with wrapped errors.